### PR TITLE
Update runbook files for lisa and legacy run.

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: microsoft/runbook/azure.yml [hunk 1]
```diff
@@ -2,14 +2,30 @@ name: azure default
...
 notifier:
   - type: html
+environment:
+  warn_as_error: false
+  allow_create: false
+  environments:
+    - nodes:
+        - type: requirement
+          core_count:
+            min: 4
```
Comment (squirrelsc) [inline microsoft/runbook/azure.yml]: leave to 2 here

File: microsoft/runbook/azure.yml [hunk 2]
```diff
@@ -2,14 +2,30 @@ name: azure default
...
+            gallery: $(gallery_image)
+            location: $(location)
 platform:
   - type: azure
     admin_private_key_file: $(admin_private_key_file)
     reserve_environment: $(reserve_environment)
     azure:
       subscription_id: $(subscription_id)
       wait_delete: false
+      availability_set_properties:
```
Comment (squirrelsc) [inline microsoft/runbook/azure.yml]: remove this

**Review by squirrelsc — CHANGES_REQUESTED** [review]

File: microsoft/runbook/tiers/t1.yml [hunk 3]
```diff
@@ -2,7 +2,17 @@ variable:
...
 testcase:
+  # v3 p0, p1 test cases
   - criteria:
       priority: [0, 1]
+  # LisaV2 P0 functional, excluding XDP, DPDK, LIS_DEPLOY
   - type: legacy
-    command: .\Run-LisaV2.ps1 -TestLocation '$(location)' -RGIdentifier '$(v2_rg)' -TestPlatform 'Azure' -ARMImageName '$(image)' -XMLSecretFile '$(v2_secret_file)' -TestPriority 0 -ResourceCleanup $(v2_rg_cleanup)
+    repo: $(v2_repo_url)
+    branch: $(v2_branch)
+    command: .\Run-LisaV2.ps1 -TestLocation '$(location)' -RGIdentifier '$(v2_rg)' -TestPlatform 'Azure' -EnableTelemetry -ARMImageName '$(gallery_image)' -XMLSecretFile '$(v2_secret_file)' -TestPriority 0 -ResourceCleanup '$(v2_rg_cleanup)' -CustomLIS '$(custom_lis)' -CustomTestParameters '$(custom_test_parameters)' -OverrideVMSize '$(override_vmsize)' -CustomKernel '$(custom_kernel)' -ExcludeTests '*DPDK*,*XDP*,^LIS-DEPLOY*,^LIS-DRIVER*' -ForceCustom -RunInParallel -TotalCountInParallel 3
```
Comment (glima) [inline microsoft/runbook/tiers/t1.yml:12]: Wait, do we have commands coming from runbooks? And hard-coded to ps1 scripts? Confused.

Comment (squirrelsc) [reply microsoft/runbook/tiers/t1.yml:12]: It's a good chance to validate below document. Could you try to find answers from it? Let me know, if any question here.
https://github.com/microsoft/lisa/blob/main/docs/run_legacy.md

Comment (glima) [reply microsoft/runbook/tiers/t1.yml:12]: I see, so we edit things on these runbooks at the pace tests are migrated/translated, then.

Comment (squirrelsc) [reply microsoft/runbook/tiers/t1.yml:12]: yes, we're simplifying and creating test standard.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**